products/models.py

```diff
@@ -0,0 +1,55 @@
+from django.db    import models
+
+class Purchase(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'purchases'
+
+class MainCategory(models.Model):
+    name        = models.URLField(max_length=45)
+    purchase_id = models.ForeignKey('Purchase', on_delete=models.CASCADE)
```

Comment:
```suggestion
    purchase = models.ForeignKey('Purchase', on_delete=models.CASCADE)
```
foreignkey로 만든 속성은 장고가 자동으로 _id를 붙혀주므로 _id 빼주세요. 
여기뿐만 아니라 다른 곳들 전부 수정해 주세요.